src/Analyzer.Core.UnitTests/Analyzer.Core.UnitTests.csproj

```diff
@@ -1,22 +1,29 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
The indentation in this file was changed from spaces to tabs.  Please revert to original formatting.